wbooks/src/app/index.tsx

```diff
@@ -10,9 +10,19 @@
 
 import React from 'react';
 import Library from '@screens/Library';
+import { SafeAreaView, StatusBar } from 'react-native';
```

Comment:
Sort imports:

* global imports first
* aliases
* local imports (`../` and `./`)

In this case:
```
import React from 'react';
import { SafeAreaView, StatusBar } from 'react-native';
import Library from '@screens/Library';

import styles from './styles';
```

Reply:
Done